ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/cv-pathfinder-custom-control-plane-policy.yml

```diff
@@ -0,0 +1,186 @@
+    sites:
+      - name: Site1
+        id: 1
+        location: one place
+
+bgp_peer_groups:
+  wan_overlay_peers:
+    password: "htm4AZe9mIQOO1uiMuGgYQ=="
+    # Overwriting TTL
+    ttl_maximum_hops: 42
+    listen_range_prefixes:
+      - 192.168.255.0/24
+
+wan_route_servers:
+  - hostname: cv-pathfinder-pathfinder
+
+wan_ipsec_profiles:
+  control_plane:
+    shared_key: ABCDEF1234567890
+  data_plane:
+    shared_key: ABCDEF1234567890666
+
+default_node_types:
+  - node_type: wan_rr
+    match_hostnames:
+      - "cv-pathfinder-custom-control-plane-policy-pathfinder.*"
+  - node_type: wan_router
+    match_hostnames:
+      - "cv-pathfinder-custom-control-plane-policy-edge.*"
+
+wan_router:
+  defaults:
+    loopback_ipv4_pool: 192.168.42.0/24
+    vtep_loopback_ipv4_pool: 192.168.255.0/24
+    filter:
+      always_include_vrfs_in_tenants: [TenantA]
+      deny_vrfs: [IT]
+  nodes:
+    - name: cv-pathfinder-custom-control-plane-policy-edge-1
+      cv_pathfinder_region: AVD_Land_West
+      cv_pathfinder_site: Site1
+      id: 1
+      l3_interfaces:
+        - name: Ethernet1
+          wan_carrier: ATT
+          wan_circuit_id: 666
+          dhcp_accept_default_route: true
+          ip_address: dhcp
+        - name: Ethernet2
+          wan_carrier: Colt
+          wan_circuit_id: 10555
+          ip_address: 172.15.5.5/31
+        - name: Ethernet3
+          wan_carrier: Comcast-5G
+          wan_circuit_id: AF830
+          ip_address: 172.20.20.20/31
+          connected_to_pathfinder: False
+
+wan_rr:
+  defaults:
+    loopback_ipv4_pool: 192.168.44.0/24
+    vtep_loopback_ipv4_pool: 192.168.144.0/24
+    data_plane_cpu_allocation_max: 1
+    bgp_as: 65000
+  nodes:
+    - name: cv-pathfinder-custom-control-plane-policy-pathfinder-1
+      id: 1
+      l3_interfaces:
+        - name: Ethernet1
+          wan_carrier: Bouygues_Telecom
+          wan_circuit_id: 777
+          static_routes:
+            - prefix: 0.0.0.0/0
+          ip_address: 10.7.7.7/31
+          peer_ip: 10.7.7.6
+        - name: Ethernet2
+          wan_carrier: Colt
+          wan_circuit_id: 10000
+          ip_address: 172.16.0.1/31
+        - name: Ethernet3
+          wan_carrier: Another-ISP
+          wan_circuit_id: 999
+          ip_address: 10.9.9.9/31
+
+wan_path_groups:
+  - name: MPLS
+    ipsec:
+      static_peers: false
+      dynamic_peers: false
+    id: 100
+  - name: INET
+    id: 101
+  - name: LTE
+    id: 102
+  - name: Equinix
+    id: 103
+  - name: Satellite
+    id: 104
+
+wan_carriers:
+  - name: Comcast
+    path_group: INET
+  - name: ATT
+    path_group: INET
+  - name: Bouygues_Telecom
+    path_group: INET
+  - name: SFR
+    path_group: INET
+  - name: Orange
+    path_group: INET
+  - name: Another-ISP
+    path_group: INET
+  - name: Colt
+    path_group: MPLS
+  - name: ATT-MPLS
+    path_group: MPLS
+  - name: Comcast-5G
+    path_group: LTE
+  - name: Inmrasat
+    path_group: Satellite
```

Comment:
lets shorten the examples - most of these values can be inherited from `CV_PATHFINDER_TESTS` so lets do it

Reply:
Done